### PR TITLE
feat: optionally compute just one index

### LINKED ISSUE #29: Option to compute just one of the speed indexes
Since these calculations are fairly expensive, it'd be valuable to avoid computing SI if we're only interested in pSI numbers. (And presumably vice versa).

@pmdartus any feelings on API for this?

### PR DESCRIPTION
**Pull request by patrickhulce**
fixes #29

### CODE REVIEW

File: test/speed-index.js [hunk 1]
```diff
@@ -94,7 +94,6 @@ test('speed index calculate the right value', async t => {
 
 	const indexes = speedIndex.calculateSpeedIndexes(data.frames, data);
 	t.is(indexes.speedIndex, 1000);
-	t.is(indexes.perceptualSpeedIndex, 1000);
```
Comment (patrickhulce) [inline test/speed-index.js:97]: this was somewhat nonsense since there we didn't actually have perceptual progress

**Review by paulirish — APPROVED** [review]
i changed the include options to be held in an enum.

otherwise this lgtm!

**Comment by paulirish** [comment]
@patrickhulce  can you merge if you're happy with my commits?

**Comment by patrickhulce** [comment]
ya looks good 👍

**Comment by paulirish** [comment]
All these PRs were shipped in 1.3.0

**Pull request merged**